source/mainnet/net/resources/release-notes.rst

```diff
@@ -11,6 +11,19 @@ Sirius Testnet
 
    Prior to Sirius, the nodes enforced that a transaction could not be deployed until 2 hours before its expiry date. With Sirius, node validation of transactions has been improved and the 2 hour window has been removed.
 
+August 24, 20222
+
+Concordium Client 4.1.0
+-----------------------
+
+- Fix bug in contract schema parsing caused by endiannes confusion.
+- Add support for smart contract schema V2. V2 schemas offer the same options as V1, but can also include a schema for the error type. This enables concordium-client to interact with contracts built using concordium-std version 4.
+
+Cargo concordium 2.1.0
+----------------------
+
+Use schemas for error values when simulating contracts. In particular support building and testing contracts with `concordium-std` version 4.
```

Comment:
```suggestion
- Fix bug in contract schema parsing caused by endiannes confusion.
- Add support for smart contract schema V2. V2 schemas offer the same options as V1, but can also include a schema for the error type. This enables `concordium-client` to interact with contracts built using `concordium-std` version 4.

Cargo concordium 2.1.0
----------------------

Use schemas for error values when simulating contracts. In particular support building and testing contracts with `concordium-std` version 4.
```
I think we should consistently "quote" `concordiu-std` and `concordium-client` so that they stand out as tool names.

Reply:
I've added it here for now so we can get this released. I will need to check the rest of the documents at a later point because in some places it is handled like code (``client``) and in others it's italicized (`client`).